charts/incubator/lldap/templates/common.yaml

```diff
@@ -0,0 +1,11 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.v1.common.loader.init" . }}
+
+{{/* Render secrets for LLDAP */}}
+{{- $secrets := include "lldap.secrets" . | fromYaml -}}
+{{- if $secrets -}}
+  {{- $_ := set .Values.secret "secret" $secrets -}}
```

Comment:
the `"secret"` is the name of the `kind: Secret` which gets expanded to fullname-secret.

So the correct fetch name becomes:
```yaml
{{- $fetchname := printf "%s-secret" $basename -}}
```